sklearn/metrics/cluster/unsupervised.py

```diff
@@ -56,6 +65,18 @@ def silhouette_score(X, labels, metric='euclidean', sample_size=None,
         <sklearn.metrics.pairwise.pairwise_distances>`. If X is the distance
         array itself, use ``metric="precomputed"``.
 
+    block_size : int, optional, default=64
```

Comment:
If you're an average user, I'm better off not making it a parameter and hard-coding something sensible! :P